blueprint.js

```diff
@@ -198,12 +198,25 @@ $Element.prototype.copy = function() {
 			res.childNodes.push(child.clone());
 		}
 	}
+	res.data = Object.assign({}, this.data);
```

Comment:
This could just be `Object.assign(res.data, this.data);`

Reply:
Good catch!